bot.py

```diff
@@ -16,17 +17,12 @@
 
 @client.event
 async def on_message(message):
-    # find message.content in commands.json and append msg with the value
-    for command in commands:
-        if message.content == command:
-            msg = commands[command]
-
-            # if the message is from sherps, add a celebratory gif
-            if message.author.id == SHERP_ID:
-                msg = "https://media.giphy.com/media/artj92V8o75VPL7AeQ/giphy.gif " + msg
-
-            await message.channel.send(msg)
-
+    # stops the bot from responding to itself
+    if message.author.bot: return
+    
+    if message.content in commands:
```

Comment:
We dont want the bot to reply to //
```suggestion
    if message.content in commands and not "//":
```